csharp/Infra.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Pulumi" Version="1.5.0-*" />
```

Comment:
@CyrusNajmabadi, does running `pulumi up` automatically restore these dependencies? (I assume we're just calling `dotnet run` which runs an implicit `dotnet restore`.)

Reply:
Yes, it does for me.  We run both "dotnet build" and "dotnet run" and both do a restore automatically.